src/main/java/net/openhft/chronicle/wire/BinaryWire.java

```diff
@@ -810,6 +831,11 @@ private StringBuilder readSpecialField(int peekCode, CharSequence keyName, int k
             case FIELD_NUMBER:
                 bytes.uncheckedReadSkipOne();
                 long fieldId = bytes.readStopBit();
+                if (fieldId == MethodReader.MESSAGE_HISTORY_METHOD_ID) {
+                    sb.setLength(0);
+                    sb.append(MethodReader.HISTORY);
```

Comment:
why is this required? generated method readers handle MESSAGE_HISTORY_METHOD_ID already

Reply:
MethodReaders will recognise the field numbers however many tools don't use generated method readers. e.g. Chronicle Reader

Reply:
Unless you give it `-m` argument. But point taken

Reply:
Could we add a comment explaining why?